src/install.c

```diff
@@ -90,8 +90,17 @@ gboolean determine_slot_states(GError **error)
 	mountlist = g_unix_mounts_get(NULL);
 	for (GList *l = mountlist; l != NULL; l = l->next) {
 		GUnixMountEntry *m = (GUnixMountEntry*)l->data;
-		g_autofree gchar *devicepath = resolve_loop_device(g_unix_mount_get_device_path(m));
-		RaucSlot *s = find_config_slot_by_device(r_context()->config,
+		g_autofree gchar *devicepath = NULL;
+		RaucSlot *s;
+
+		/* We can ignore internal devices such as tmpfs, cgroups, etc. here.
+		 * They won't start with a '/', thus we simply use this for
+		 * checking */
+		if (g_unix_mount_get_device_path(m)[0] != '/')
```

Comment:
Can this string be empty or NULL?

Reply:
Mh.. from the implementation I fear we cannot be sure that it will not become NULL. However, empty string should be '\0' and thus not be an issue here, right?

Reply:
Pushed a fixup addressing this.